components/brave_wallet_ui/components/desktop/views/portfolio/components/token-lists/token-list.tsx

```diff
@@ -555,7 +564,7 @@ export const TokenLists = ({
                 placeholder={getLocale('braveWalletSearchText')}
                 action={onSearchValueChange}
                 value={searchValue}
-                isV2={true}
+                isV2={isV2}
```

Comment:
Please pass `isV2={true}` from the `Portfolio Overview` page, or it will show the wrong `search bar` style.

Example from this Test Plan video:

![Screenshot 25](https://github.com/brave/brave-core/assets/40611140/fb445509-890f-450a-b1cb-567fd94a6c86)

What it should be:

![Screenshot 24](https://github.com/brave/brave-core/assets/40611140/db17006a-a220-4001-9248-3c68c41a1270)

Reply:
@Douglashdaniel Good catch. Fixed here: f649e386b62d6e0fc9aa5b8f9bf374a7adcfe685